tests/api/DisplayEventsCest.php

```diff
@@ -7,7 +7,7 @@ public function _before(ApiTester $I)
     }
 
     public function createEvents(ApiTester $I)
-    {   
+    {
```

Comment:
git checkout main tests/api/DisplayEventsCest.php

Don't bring this change to the pull request